cpp/benchmarks/io/parquet/parquet_reader_options.cpp

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <benchmarks/common/generate_input.hpp>
+#include <benchmarks/fixture/benchmark_fixture.hpp>
+#include <benchmarks/fixture/rmm_pool_raii.hpp>
+#include <benchmarks/io/cuio_common.hpp>
+#include <benchmarks/io/nvbench_helpers.hpp>
+
+#include <cudf/io/parquet.hpp>
+#include <cudf/utilities/default_stream.hpp>
+
+#include <nvbench/nvbench.cuh>
+
+constexpr size_t data_size = 512 << 20;
+
+std::vector<std::string> get_col_names(cudf::io::source_info const& source)
+{
+  cudf::io::parquet_reader_options const read_options =
+    cudf::io::parquet_reader_options::builder(source);
+  return cudf::io::read_parquet(read_options).metadata.column_names;
+}
+
+template <column_selection ColSelection, row_selection RowSelection, cudf::type_id Timestamp>
+void BM_parquet_read_options(nvbench::state& state,
+                             nvbench::type_list<nvbench::enum_type<ColSelection>,
+                                                nvbench::enum_type<RowSelection>,
+                                                nvbench::enum_type<Timestamp>>)
+{
+  cudf::rmm_pool_raii rmm_pool;
+
+  auto constexpr num_chunks          = 1;
+  auto constexpr str_to_categories   = true;
```

Comment:
The group of benchmarks that varied these options got removed by accident along with num_rows: https://github.com/rapidsai/cudf/commit/d39b957fea7a2d4b501a88ca0ee5c879c2f22067#diff-2beb3d20a5e5999eee1cc7cdbac4a9825480d2e08090518974a7b0f05e708d78
I think we should restore this group "misc_options". Timestamp can be places there instead of being bundled with column selection.

Reply:
Done